Lombiq.Tests.UI/Extensions/BasicOrchardFeaturesTestingUITestContextExtensions.cs

```diff
@@ -506,6 +508,59 @@ public static Task TestRegistrationAsync(this UITestContext context, UserRegistr
                     .SearchForFeature(featureName).IsEnabled.Should.Equal(originalEnabledState));
             });
 
+    public static Task TestMediaOperationsAsync(this UITestContext context) =>
+        context.ExecuteTestAsync(
+            "Test media operations",
+            async () =>
+            {
+                await context.GoToRelativeUrlAsync("Admin/Media");
+
+                context.UploadSamplePngByIdOfAnyVisibility("fileupload");
+                context.UploadSamplePdfByIdOfAnyVisibility("fileupload");
+
+                // Workaround for pending uploads.
```

Comment:
Document why this workaround is necessary (or why is it a workaround in the first place and what would be the ideal solution).